src/views/Project/Details/Users/ProjectUsergroupList/index.js

```diff
@@ -113,13 +125,27 @@ export default class ProjectUsergroupList extends React.PureComponent {
                     <Actions
                         projectId={this.props.projectId}
                         readOnly={this.props.readOnly}
+                        activeUserRole={
+                            this.getActiveUserRole(
+                                this.props.projectRoleList,
+                                this.props.memberships,
+                                this.props.activeUser.userId,
+                            )}
                         row={row}
                     />
                 ),
             },
         ];
     }
 
+    getActiveUserRole = memoize((projectRoleList, memberships, memberId) => (
```

Comment:
What if the return value of find is `undefined`?

Reply:
It is filtered.

Reply:
what if
```
            p => p.id === memberships.find(
                m => m.member === memberId,
            )
```
returns undefined